mavisp/core.py

```diff
@@ -217,11 +220,15 @@ def ingest(self, stop_at='critical'):
             mutations = r['mutations']
             curators = r['curators']
             uniprot_ac= r["uniprot_ac"]
-            refseq_d= r["refseq_d"]
+            refseq_id= r["refseq_id"]
             if mutations is None:
                 mavisp_criticals.append(MAVISpCriticalError("the mutation list was not available, readable or in the expected format"))
             if curators is None:
                 mavisp_criticals.append(MAVISpCriticalError("the metadata file was not available, readable or in the expected format"))
```

Comment:
since we are now checking for multiple fields, we can tailor our error messages accordingly